src/pages/WizzardPage3/WizzardPage3.module.scss

```diff
@@ -0,0 +1,50 @@
+.form {
+  background-color: white;
+  font-weight: 600;
+}
+.input_field {
+  padding: 20px;
+  max-height: 40vh;
+  overflow-y: scroll;
+}
+.input_field::-webkit-scrollbar {
+  width: 5px;
+  background-color: rgba(0, 0, 0, 0.693);
+}
+
+.input_field::-webkit-scrollbar-thumb {
+  background: white;
+  border-radius: 25px;
+  border: 1px solid black;
+}
+
+.textarea_agri {
+  width: 100%;
+
+  textarea {
```

Comment:
![image](https://user-images.githubusercontent.com/82144052/189626702-aaad8ebc-e015-41b6-b582-0b2a044ca46c.png)

You can limit the textarea so the user can't widen it.